Dependencies/doorstop_config.ini

```diff
@@ -0,0 +1,3 @@
+[UnityDoorstop]
+enabled=true
+targetAssembly=QModManager/QModInstaller.dll
```

Comment:
If this means that players can't simply verify the integrity of their game files to uninstall QMM,  
then how are they going to uninstall or disable it at all?

What about people who switch between Nitrox and QMM?  
What will it look like when a player wants to disable all mods for one reason or another?

Reply:
Manual uninstallation from add & remove programs or this: 

![Screenshot_2019-05-09-09-45-32](https://user-images.githubusercontent.com/32238504/57570285-e3af7d00-7408-11e9-97a9-b7cc0fa8c044.jpg)

Reply:
Uninstalling QModManager by verifying game files was never intentional, it was just a side effect of how the injection was done. 
A file verification would cause QModManager to appear installed in the programs list, but not patched.